src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
```

Comment:
It seems that you are selecting the first 'tr' element and the 'tbody' without ensuring they exist on the page. It would be safer to add a check to ensure the elements are not null before adding event listeners or performing operations on them.